card.py

```diff
@@ -196,6 +196,7 @@ def check_if_stale(issue, label_name, warn_days_allowed, error_days_allowed, ass
             #     check_column_label(labels, 'completed', issue)
             if column is COLUMNS.IMPEDED:
                 check_column_label(labels, 'impeded', issue)
+                check_if_stale(issue, 'impeded', 7, 28, assigned)
```

Comment:
does this actually check if the ticket has been updated by a comment as was suggested in planning? or have some other way of checking that the ticket has been acknowleged? It appears that this check only takes into account creation time and whether a ticket has moved recently, which does not fulfil the acceptence criteria of having a way to acknowledge that the ticket is _still_ impeded.

Reply:
Has this been resolved?